### ISSUE TITLE
About crowdpose results in crowdpose.md page.

### ISSUE THREAD

**Issue by ArchNew** [MVIG-SJTU/AlphaPose#714]
According to https://github.com/MVIG-SJTU/AlphaPose/blob/master/docs/CrowdPose.md, the results they compared are on crowdpose validation set. However, the same results they reported in the paper, it shows that those results are on crowdpose test set. I'm confused.

I tried to reproduce the results of simple baselines on crowdpose. But even I used the ground truth bounding box, the results on crowdpose test are nowhere near the reported in the Paper. However, if the results reported are on the validation set, it makes a lot of sense.

**Comment by mobeixiaoxin** [comment]
@ArchNew ,How to distinguish the training, validation and test data of crowdpose dataset? The dataset I downloaded has only one image directory?
 Looking forward to your reply, thank you

**Comment by ArchNew** [comment]
> @ArchNew ,How to distinguish the training, validation and test data of crowdpose dataset? The dataset I downloaded has only one image directory?
> Looking forward to your reply, thank you

I'm sorry to reply after so long. According to the author, they evaluate on valset.
